etc/picongpu/cpuNumaStarter.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
can you please add a license header **and** a description what this script does? :)